versioned/storage/common/src/test/java/org/projectnessie/versioned/storage/common/logic/TestIndexesLogicImpl.java

```diff
@@ -76,7 +78,7 @@ public void completeIndexesInCommitChain(boolean discrete) throws Exception {
     }
     ObjId secondaryHeadId = secondary.get(0);
     CommitObj secondaryHead = commitLogic.fetchCommit(secondaryHeadId);
-    soft.assertThat(secondaryHead).isNotNull();
+    assertThat(secondaryHead).isNotNull();
```

Comment:
Better leave the `soft.assertThat()` and add a `soft.assertAll()` after it.
Otherwise previous soft-assertions would get lost. (Same below)

Reply:
That was to calm down IntelliJ that cannot understand soft assertions. But OK, I will revert.